d3a_api_client/rest_market.py

```diff
@@ -12,23 +12,19 @@
 
 class RestMarketClient(RestCommunicationMixin):
 
-    def __init__(self, simulation_id, device_id, domain_name,
+    def __init__(self, simulation_id, area_id, domain_name,
                  websockets_domain_name):
         self.simulation_id = simulation_id
-        self.device_id = device_id
+        self.device_id = area_id
         self.domain_name = domain_name
         self.jwt_token = retrieve_jwt_key_from_server(domain_name)
 
         self.dispatcher = WebsocketMessageReceiver(self)
-        self.websocket_thread = WebsocketThread(simulation_id, device_id, self.jwt_token,
+        self.websocket_thread = WebsocketThread(simulation_id, area_id, self.jwt_token,
```

Comment:
Please notify Colin for this change in order to be able to adapt his scripts accordingly.